docs/Reference/Routes.md

```diff
@@ -653,3 +653,18 @@ fastify.route({
   }
 })
 ```
+
+### ⚠  HTTP version check
+
+Fastify will check the HTTP version of every request, based on configuration options ([http2](./Server.md#http2), [https](./Server.md#https), and [serverFactory](./Server.md#serverfactory)), to determine if it matches one or all of the following versions: `2.0`, `1.1`, and `1.0`. If Fastify receives a different HTTP version in the request it will return a `505 HTTP Version Not Supported` error.
+
+|                          | 2.0 | 1.1 | 1.0 | skip |
+|--------------------------|-----|-----|-----|------|
+| http2                    |  ✓  |     |     |      |
+| http2 + https            |  ✓  |     |     |      |
+| http2 + https.allowHTTP1 |  ✓  |  ✓  |  ✓  |      |
+| https                    |     |  ✓  |  ✓  |      |
+| http                     |     |  ✓  |  ✓  |      |
+| serverFactory            |     |     |     |   ✓  |
+
+Note: The internal HTTP version check will be removed in future when the Node.js implemented this [feature](https://github.com/nodejs/node/issues/43115).
```

Comment:
Please hard wrap these lines to be consistent with the rest of the document.

Reply:
Done

Reply:
No, I did not mean to make it a block quote. I meant wrap the lines at 80 characters with a line feed.